packages/neuron-wallet/src/block-sync-renderer/sync/light-connector.ts

```diff
@@ -92,7 +92,11 @@ export default class LightConnector extends Connector<CKBComponents.Hash> {
   private async fetchDepCell() {
     const depGroupOutputsData: string[] = await this.getDepTxs()
     const depGroupTxHashes = [
-      ...new Set(depGroupOutputsData.map(v => unpackGroup.unpack(v).map(v => v.tx_hash.toHexString())).flat()),
+      ...new Set(
+        depGroupOutputsData
+          .map(v => unpackGroup.unpack(v).map(v => `0x${v.tx_hash.toString(16).padStart(64, '0')}`))
```

Comment:
In what case `tx hash` should be padded

Reply:
I use wrong type with Lumos, https://github.com/nervosnetwork/neuron/blob/develop/packages/neuron-wallet/src/block-sync-renderer/sync/light-connector.ts#L28-L36 , I will change these to `const unpackGroup = molecule.vector(blockchain.OutPoint)`